modules/jupyter_iap/iap.tf

```diff
@@ -37,7 +37,7 @@ resource "helm_release" "iap_jupyter" {
   chart            = "${path.module}/charts/iap_jupyter/"
   namespace        = var.namespace
   create_namespace = !contains(data.kubernetes_all_namespaces.allns.namespaces, var.namespace)
-
+  timeout          = 1200
```

Comment:
what is this for?

Reply:
maybe leave a comment explaining why it's needed

Reply:
helm install takes time.. because of autopilot pod scheduling may take addtional time.. 

and if pods are not running within 5 minutes tf apply will fail